apps/fishing-map/features/resources/resources.slice.ts

```diff
@@ -3,9 +3,11 @@ import {
   ResourcesState as CommonResourcesState,
   resourcesSlice,
 } from '@globalfishingwatch/dataviews-client'
-import { ThinningLevels, THINNING_LEVELS } from 'data/config'
+import {  ThinningConfig } from '@globalfishingwatch/api-types'
+import { THINNING_LEVEL_BY_ZOOM, THINNING_LEVEL_ZOOMS } from 'data/config'
 import { selectDebugOptions } from 'features/debug/debug.slice'
 import { isGuestUser } from 'features/user/user.slice'
+import { selectUrlMapZoomQuery } from 'routes/routes.selectors'
```

Comment:
this means we are going to read the value debounced by [1 second](https://github.com/GlobalFishingWatch/frontend/blob/fishing-map/tracks-thinning-zoom-level/apps/fishing-map/features/timebar/timebar.hooks.ts#L58)
but there I can't figure out a direct way of reading atoms into selectors... so any idea?

Reply:
I actually think this is a feature not a bug - and I relied on the same thing for loading chunks by years. Waits a bit for the user to "settle" on a zoom level/time before sending the requests.